rws/rws-be/pom.xml

```diff
@@ -10,6 +10,20 @@
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>rws-be</artifactId>
+    <dependencies>
+        <dependency>
+            <groupId>com.vertex</groupId>
+            <artifactId>database</artifactId>
+            <version>1.0-SNAPSHOT</version>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
```

Comment:
You don't need this dependency. Parent pom.xml contains "sping-boot-starter-test"